cli/main.go

```diff
@@ -93,7 +96,16 @@ func start(_ context.Context, cmd *cli.Command) error {
 		log.Error("no relay passed the health-check!")
 	}
 
-	log.Infof("Listening on %v", listenAddr)
+	if metricsEnabled {
+		go func() {
```

Comment:
I'd omitted graceful shutdown from this PR given i didn't know how much traction it'd get - do we think we'd require it for the initial implementation here?

Reply:
I don't think it's necessary for this PR. Let's save that for later.